### PR TITLE
Bblfsh web

### PR DESCRIPTION
**Pull request by smacker**
This PR adds bblfsh-web into superset.

It also contains simple Makefile to apply changes on superset.

*Note*:
It is **not** the way to go for us, in my opinion.
I have just copied original files to see the diff for now.
It means updates for that files must be done manually.
In theory, we can do something with `git filter-branch` or `git checkout -- path` with another remote or somehow else implement `git merge` for those files only.
Though it's out of the scope of this PR.

It looks like this after `make build` && `docker-compose up`
Screenshot 2019-04-30 at 17 54 43

### CODE REVIEW

**Comment by se7entyse7en** [comment]
It looks really good!

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+		cp -r "$(PATCH_SOURCE_DIR)/$${file}" "$(SUPERSET_DIR)/$${file}"; \
+	done; \
+
+# Overrides files in the superset repository using symlinks
+.PHONY: patch-dev
+patch-dev:
+	@for file in $(ADD_FILES) $(OVERRIDE_FILES); do \
+		echo "patching $${file}"; \
+		rm -rf "$(SUPERSET_DIR)/$${file}"; \
+		ln -s "$(PATCH_SOURCE_DIR)/$${file}" "$(SUPERSET_DIR)/$${file}"; \
```
Comment (se7entyse7en) [inline Makefile]: Why `cp` in one case and `ln` in the other?

Comment (smacker) [reply Makefile]: we must `cp` to build docker image. Or `build` will fail.

`ln` is for development. So you can run `FLASK_ENV=development flask run -p 8088 --with-threads --reload --debugger` with `npm run dev-server` in `superset` dir locally and have auto-reload when you change the files.

Comment (se7entyse7en) [reply Makefile]: Great! Thanks for the explanation

**Pull request merged**